ipyleaflet/leaflet.py

```diff
@@ -2035,29 +2035,29 @@ def remove_legend_element(self, key):
 class ColormapControl(WidgetControl):
     """ColormapControl class, with WidgetControl as parent class.
 
-    A control which contains a colormap, to be used with Choropleth.
+    A control which contains a colormap.
 
     Attributes
     ----------
     caption : str, default 'caption'
         The caption of the colormap.
-    colormap_choice : str, default 'linear.YlOrRd_04'
-        The choosen colormap.
+    colormap: branca.colormap.ColorMap instance, default None
+        The colormap used for the effect.
     value_min : float, default 0.0
         The minimal value taken by the data to be represented by the colormap.
     value_max : float, default 1.0
         The maximal value taken by the data to be represented by the colormap.
     """
     caption = Unicode('caption')
-    colormap_choice = Any(linear.YlOrRd_04)
+    colormap = Any(linear.YlOrRd_04)
```

Comment:
Could be interesting to use traitlets' [Instance](https://traitlets.readthedocs.io/en/stable/trait_types.html#traitlets.Instance)?

Reply:
Thanks for the comment. Traitlet type and default value in the doc string have been modified in commit